`debug/test` build failure.

Repro steps:

```bash
cd debug/test
make -j16
```

Expected: 

Build should pass.

Actual:

```
[100%] Built target test_style_transfer_data_iterator.cxxtest
/Users/schhabra/workspace/turicreate/test/unity/toolkits/object_detection/test_object_detector.cxx:315:58: error: allocating an object of abstract class type 'turi::object_detection::(anonymous namespace)::mock_compute_context'
  std::unique_ptr<mock_compute_context> mock_context(new mock_compute_context);
                                                         ^
/Users/schhabra/workspace/turicreate/src/ml/neural_net/compute_context.hpp:116:42: note: unimplemented pure virtual method 'create_style_transfer' in 'mock_compute_context'
  virtual std::unique_ptr<model_backend> create_style_transfer(
                                         ^
/Users/schhabra/workspace/turicreate/test/unity/toolkits/object_detection/test_object_detector.cxx:547:58: error: allocating an object of abstract class type 'turi::object_detection::(anonymous namespace)::mock_compute_context'
  std::unique_ptr<mock_compute_context> mock_context(new mock_compute_context);
                                                         ^
/Users/schhabra/workspace/turicreate/test/unity/toolkits/object_detection/test_object_detector.cxx:789:58: error: allocating an object of abstract class type 'turi::object_detection::(anonymous namespace)::mock_compute_context'
  std::unique_ptr<mock_compute_context> mock_context(new mock_compute_context);
                                                         ^
/Users/schhabra/workspace/turicreate/test/unity/toolkits/object_detection/test_object_detector.cxx:936:26: error: allocating an object of abstract class type 'turi::object_detection::(anonymous namespace)::mock_compute_context'
  mock_context.reset(new mock_compute_context);
                         ^
4 errors generated.
make[2]: *** [test/unity/toolkits/object_detection/CMakeFiles/test_object_detector.cxxtest.dir/test_object_detector.cxx.o] Error 1
make[1]: *** [test/unity/toolkits/object_detection/CMakeFiles/test_object_detector.cxxtest.dir/all] Error 2
make: *** [all] Error 2
```

--- a/test/unity/toolkits/object_detection/test_object_detector.cxx
+++ b/test/unity/toolkits/object_detection/test_object_detector.cxx
@@ -202,6 +202,13 @@ public:
     return nullptr;
   }
 
+  std::unique_ptr<model_backend> create_style_transfer(
+      const float_array_map& config,
+      const float_array_map& weights) override
+  {
+    return nullptr;
+  }
+
   mutable std::deque<create_augmenter_call> create_augmenter_calls_;
   mutable std::deque<create_object_detector_call> create_object_detector_calls_;
 };

